src/renderer/components/TranscriptionBlock.tsx

```diff
@@ -1,16 +1,30 @@
 import styled from '@emotion/styled';
-import { Box, colors } from '@mui/material';
+import { Box } from '@mui/material';
 import { Transcription } from 'sharedTypes';
+import colors from '../colors';
 
 const TranscriptionBox = styled(Box)`
-  background: ${colors.common.white};
+  background: ${colors.grey[700]};
+  border-radius: 5px;
+  color: ${colors.grey[300]};
+  overflow-y: scroll;
   height: 100%;
   padding: 20px;
+
+  ::-webkit-scrollbar {
+    width: 3px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    background: ${colors.yellow[500]};
+  }
 `;
 
 const Word = styled('span')`
   &:hover {
-    background: orange;
+    color: ${colors.grey['000']};
+    background: rgba(255, 187, 85, 0.5);
```

Comment:
I just confirmed with @emmapainter that this color code was an error in the designs. It should be yellow 500 at 50% opacity.

Reply:
I believe the new line can be written as `rgba(colors.yellow[500], 0.5)`

Reply:
I tried that too but couldn't get it to work. According to [this](https://stackoverflow.com/questions/7015302/css-hexadecimal-rgba) very old SO thread 8bit hex codes were going to be supported. (where the last 2 bits represent opacity). 

This `background: ${colors.yellow[500]}80;` seems to work but feels wrong tbh...

I can't find any official W3C resources that confirm its supported.